app/javascript/components/filters/Filters.vue

```diff
@@ -5,7 +5,8 @@
       :name="filter.name"
       :title="filter.title" 
       :options="filter.options"
-      :type="filter.type">
+      :type="filter.type"
+      :presetFilterOptions="presetOptionsForFilterType(filter.name)">
```

Comment:
The row above reveals that `filter` has a key of `type`, and then your method uses the word `Type` but then passes in the name, which might be a little confusing

Reply:
changed method to presetOptionsForFilter